src/app/client/src/app/modules/certificate/components/certificate-configuration/certificate-configuration.component.ts

```diff
@@ -167,19 +167,25 @@ export class CertificateConfigurationComponent implements OnInit, OnDestroy {
    */
   getTemplateList() {
     const request = {
-      request: {
-        orgId: _.get(this.userService, 'userProfile.rootOrgId'),
-        key: 'certList'
+      'request': {
+          'filters': {
+              'certType': 'cert template layout',
+              'channel': this.userService.channel,
+              'mediaType': 'image'
+          },
+          'fields': ['indentifier', 'name', 'code', 'certType', 'data', 'issuer', 'signatoryList', 'artifactUrl', 'primaryCategory', 'channel'],
+          'limit': 100
       }
-    };
-
-    return this.uploadCertificateService.getCertificates().pipe(
+  };
+    return this.certRegService.getCertLayouts(request).pipe(
       tap((certTemplateData) => {
         const templatList = _.get(certTemplateData, 'result.content');
         templatList.forEach(templat => {
           this.certTemplateList.push(templat);
-          if(this.templateIdentifier && (templat.identifier == this.templateIdentifier)) {
+          if (this.templateIdentifier && (templat.identifier === this.templateIdentifier)) {
```

Comment:
Use the index login instead of forEach. 
Get the index of the items by (identifier) & move that item to top.